llvm/utils/filecheck_lint/filecheck_lint.py

```diff
@@ -78,32 +78,41 @@ def levenshtein(s1: str, s2: str) -> int:  # pylint: disable=g-doc-args
 
 
 class FileRange:
-    """Stores the coordinates of a span on a single line within a file.
+    """Derives the coordinates of a span on a single line within a file.
 
     Attributes:
-      line:         the line number
-      start_column: the (inclusive) column where the span starts
-      end_column:   the (inclusive) column where the span ends
+      content:      line str
+      start_byte: the (inclusive) byte offset the span starts
+      end_byte:   the (inclusive) byte offset the span ends
     """
 
-    line: int
-    start_column: int
-    end_column: int
+    content: str
+    start_byte: int
+    end_byte: int
 
     def __init__(
         self, content: str, start_byte: int, end_byte: int
     ):  # pylint: disable=g-doc-args
-        """Derives a span's coordinates based on a string and start/end bytes.
```

Comment:
We'll want to keep a 1-liner header for the documentation of this constructor, as previously.

How about `Stores the coordinates of a span based on a string and start/end bytes.`?

Reply:
Why comment even needed for constructor? There already doc comment for class, so i moved it into class.

Reply:
This is the [authoritative documentation](https://peps.python.org/pep-0257/#what-is-a-docstring) we [want to follow](https://www.llvm.org/docs/CodingStandards.html#python-version-and-source-code-formatting).

Beyond that though, the docstring of the constructor still actually provides additional information for how it is supposed to be used (hence there is useful information in the docstring), but does not have a summary line---which is non-standard. Please fix.